client/src/app/containers/Game/useGame.ts

```diff
@@ -145,10 +145,11 @@ export function useGame(): Result {
     const { currentUser } = useAuth();
     const { setAlert } = useAlert();
     const dispatch = useDispatch();
+    const base = `https://lambda.hobbycribbage.com`;
```

Comment:
🔮 In the future, it may be prudent to upload a `settings.json` as part of our SPA bundle that we can independently update without making a code change.

Then again, making a code change on a hobby project is just as fast so 🤷

Reply:
I agree. I don't know how to do that. What's the trick?

Reply:
It's a bit more work than this (obviously), and it's been a while since I've done this, but generally:
1. We'd have to know the name of the bucket from which the CDN distributes our bundle (CDK would give us this)
2. We'd have to upload `settings.json` to that bucket, as a sibling to our `index.html`
3. We'd have to handle uploading that with our deploy -- in the past, I've used a separate `ts-node` script and a `postdeploy` npm script in our CDK package to do this (notes about [post*](https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts) scripts)
4. We'd also want to tell our CDN to invalidate the cache for `settings.json` so our clients get the updated version
5. Ideally, we'd also want an easy way to upload something there independently of our deploy since that's the whole point of this

This smells like a separate PR to me.